backend/game_engine/common.py

```diff
@@ -187,7 +194,18 @@ def _to_dict_item(self, v):
                 'Serializable object contains unsupported attribute {}'.format(v))
 
     def to_dict(self):
-        log_message('to_dict called for {}'.format(self.__class__))
+        def massage_for_logging(attrs_dict):
+            if hasattr(self, 'recipient_phone_numbers'):
+                #sometimes phone_numbers is not a list
```

Comment:
nit: please format comments in accordance with PEP 8 style guide

https://www.python.org/dev/peps/pep-0008/#block-comments